src/iocore/cache/CMakeLists.txt

```diff
@@ -48,8 +48,8 @@ target_include_directories(inkcache PRIVATE ${CMAKE_SOURCE_DIR}/lib)
 
 target_link_libraries(
   inkcache
-  PUBLIC ts::aio ts::hdrs ts::inkevent ts::tscore
-  PRIVATE ts::proxy fastlz ZLIB::ZLIB
+  PUBLIC ts::aio ts::hdrs ts::inkevent ts::tscore ts::tsapibackend
+  PRIVATE fastlz ZLIB::ZLIB
```

Comment:
$ grep -Ir "#include \"api" src/iocore/cache/
src/iocore/cache/HttpTransactCache.cc:#include "api/APIHook.h"
src/iocore/cache/HttpTransactCache.cc:#include "api/InkAPIInternal.h"
src/iocore/cache/unit_tests/stub.cc:#include "api/InkAPIInternal.h"

Is the change to PUBLIC intentional? It's surprising.